### ISSUE TITLE
Clarify that the OpenJS Foundation cannot make a project "Closed Source"

### ISSUE THREAD

**Issue by mcollina** [openjs-foundation/cross-project-council#1115]
A maintainer gave me this objection, and I gave them this answer:

> The OpenJS Foundation cannot make a project Closed source because the community behind the project would just fork it at the latest released version, making such a "closed source" fork moot.

We might need better language to describe these F.A.Q.

**Comment by tobie** [comment]
Yeah, it would be valuable to put down in writing exactly what IP the Foundation actually owns: it's actually just the trademarks.

Copyrights and patents remain the property of their authors who are licensing their contributions as open source under the terms of the project's license.

So the only thing that the foundation could actually do would be to disallow a fork to keep its current name. That's really it.
